dev/tools/deploy-to-kube

```diff
@@ -91,7 +91,10 @@ def main(args):
 
             service_name = os.path.basename(root)
             image_prefix = utils.get_image_prefix(args)
-            image_tag = utils.get_image_tag()
+            if args.image_tag:
```

Comment:
Not a blocker, but I'm wondering if we should have e.g. `args.image_tag()`, and if you specify `image_tag` then `image_tag()` returns that value.

`args` becomes more of a configuration object.